web/src/user_group_edit_members.ts

```diff
@@ -124,7 +144,13 @@ function show_user_group_membership_request_result({
     remove_class,
     already_added_users,
     ignored_deactivated_users,
-}) {
+}: {
+    message: string;
+    add_class: string;
+    remove_class: string;
```

Comment:
Since this can never be undefined, we can remove the conditionals in the function that checks whether these two strings exist in a follow-up.

Reply:
Added as a bonus commit. :+1: